flepimop/gempyor_pkg/src/gempyor/compartments.py

```diff
@@ -785,7 +853,14 @@ def parse_parameter_strings_to_numpy_arrays(
 
         return rc
 
-    def get_compartments_explicitDF(self):
+    def get_compartments_explicitDF(self) -> pd.DataFrame:
+        """
+        Returns a copy of the compartments information DataFrame.
+        All columns receive a 'mc_' prefix.
```

Comment:
Header lines should be one sentence, unless the "All columns..." is a description in which case a newline would be good for clarity.